deformdemo/test.py

```diff
@@ -2736,6 +2736,16 @@ def test_submit_filled(self):
         # py2/py3 compat, py2 adds extra u prefix
         self.assertTrue("bar" in text)
 
+    def test_special_chars(self):
+        findid('deformField1').send_keys('foo')
+        self.assertTrue(findxpath('//p[text()="foo & bar"]').is_displayed())
```

Comment:
When I type `foo & bar` in the input field, no text is displayed in the popup, so this test should fail, but it appears to pass.